src/library_webgl.js

```diff
@@ -2661,7 +2667,8 @@ var LibraryGL = {
       // hoist the heap out of the loop for size and for pthreads+growth.
       var heap = HEAPF32;
       value >>= 2;
-      for (var i = 0; i < 4 * count; i += 4) {
+      count <<= 2;
+      for (var i = 0; i < count; i += 4) {
```

Comment:
I suspect this just increases code size, as once the JIT seems `count`'s type it should LICM the multiplication (and turn it into a shift)?

Reply:
Yeah, I was thinking that in here. Given that we don't have a view into browsers JITs to verify that it happens, and we'd have to just trust multiple browsers getting that optimization right, it is not appealing to rely on.

The code size increase is 6-4=2 bytes, and this is in a legacy path (e.g. people targeting old browsers), which is why this seems worth it.

Reply:
Sounds reasonable.